### PR TITLE
configurable number of approvals required

### PR DESCRIPTION
**Pull request by niij**
I plan to upstream this so we don't have to maintain our own fork of DataDog's Slapr workflow

### CODE REVIEW

File: slapr/__main__.py [hunk 1]
```diff
@@ -19,11 +19,12 @@
...
     slapr_bot_user_id=os.environ["SLAPR_BOT_USER_ID"],
+    number_of_approvals_required=max(1, int(os.environ.get("SLAPR_NUMBER_OF_APPROVALS_REQUIRED", 1))),
     emoji_review_started=os.environ.get("SLAPR_EMOJI_REVIEW_STARTED", "review_started"),
-    emoji_approved=os.environ.get("SLAPR_EMOJI_APPROVED", "tests-passed"),
+    emoji_approved=os.environ.get("SLAPR_EMOJI_APPROVED", "approved"),
     emoji_needs_change=os.environ.get("SLAPR_EMOJI_CHANGES_REQUESTED", "changes_requested"),
-    emoji_merged=os.environ.get("SLAPR_EMOJI_MERGED", "merged-purple"),
-    emoji_closed=os.environ.get("SLAPR_EMOJI_CLOSED", "rip"),
+    emoji_merged=os.environ.get("SLAPR_EMOJI_MERGED", "merged"),
+    emoji_closed=os.environ.get("SLAPR_EMOJI_CLOSED", "closed"),
```
Comment (niij) [inline slapr/__main__.py:27]: Our Banno repos are using custom emojis for these 3. In all the Banno workflows I've seen the env vars match what's here so reverting these 3 lines should be safe.

I plan to upstream these changes back into the DataDog/slapr repo so we don't have to maintain our own fork

**Comment by niij** [comment]
@Banno/team-mirai

**Review by tbrown1979 — APPROVED** [review]
Looks great!! Thanks for upstreaming this as well 😄

**Comment by niij** [comment]
tested and working

**Pull request merged**